README.md

```diff
@@ -108,6 +108,16 @@ Performance comparison:
 ( _zsh_nvm_lazy_load; )  0.01s user 0.01s system 168% cpu 0.012 total
 ```
 
+#### Extra commands to trigger lazy loading
+By default lazy loading nvm is triggered by running the `nvm`, `node`, `npm` commands or any installed npm global binaries.
+If you want to trigger the lazy loading via extra arbitrary commands you can define `NVM_LAZY_LOAD_EXTRA_COMMANDS` and set it to an array of commands as strings.
```

Comment:
Maybe add a sentence explaining why this is useful?

e.g: This is useful if you run a command that expects to have Node.js in it's path.

Reply:
Done!